.travis.yml

```diff
@@ -4,10 +4,11 @@ node_js:
   - "7"
   - "8"
 before_install:
-  - wget http://git.savannah.gnu.org/cgit/parallel.git/plain/src/parallel # Minimal installation
-  - chmod 755 parallel
+  - # Minimal installation as desdcribed in https://github.com/martinda/gnu-parallel
+  - wget http://git.savannah.gnu.org/cgit/parallel.git/plain/src/parallel
+  - chmod 700 parallel # Need rwx permissoins for user.
```

Comment:
Okay. I just put read/write permissions in case the system needed to alter files to execute use parallel.